README.md

```diff
@@ -0,0 +1,127 @@
+        <br />
+        <a href="https://github.com/rarity-adventure/rarity-frontend"><strong>Explore the docs »</strong></a>
+        <br />
+        <a href="https://rarity.game/">View Demo</a>
+        ·
+        <a href="https://github.com/rarity-adventure/rarity-frontend/issues">Report Bug</a>
+        ·
+        <a href="https://github.com/rarity-adventure/rarity-frontend/issues">Request Feature</a>
+    </p>
+  </div>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+-   [About the Project](#about-the-project)
+    -   [Built With](#built-with)
+-   [Getting Started](#getting-started)
+-   [License](#license)
+-   [Acknowledgements](#acknowledgements)
+-   [Contributions](#contributions)
+
+<br />
+
+<!-- ABOUT THE PROJECT -->
+<h2 align="center"> About The Project </h2>
+
+<p>Rarity frontend provides an interface for players to easily interact with the Rarity game. Rarity is achieved via an active economy, summoners must level, gain feats, learn spells, to be able to craft gear. This allows for market driven rarity while allowing an ever growing economy. Feats, spells, and summoner gear is ommitted as part of further expansions.</p>
+
+<br />
+
+### Built With
+
+-   [ReactJS](https://reactjs.org/)
+-   [TypeScript](https://www.typescriptlang.org/)
+-   [Apollo](https://www.apollographql.com/)
+
+<br />
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This is an example of how you can set up your project locally. To get a local copy up and running follow these simple example steps.
+
+1. Clone the repo
+
+```sh
+git clone with HTTPS          https://github.com/rarity-adventure/rarity-frontend.git
+git clone with SSH            git@github.com:rarity-adventure/rarity-frontend.git
+git clone with Github CLI     gh repo clone rarity-adventure/rarity-frontend
+```
+
+2. Development
+
+```sh
+1. npm install --force       # Force install dependencies for project
+2. npm run start             # Starts the app
+```
+
+3. Setting up Metamask
+
+### [Mainnet](https://docs.fantom.foundation/tutorials/set-up-metamask)
+
+```sh
+Network Name:       Fantom Opera
+RPC URL:            htps://rpc.ftm.tools/
+ChainID:            250
+Symbol:             FTM
+Block Explorer:     https://ftmscan.com/
+```
+
+### [Testnet](https://docs.fantom.foundation/tutorials/set-up-metamask-testnet)
+
+```sh
+Network Name:       Fantom testnet
+New RPC Url:        https://rpc.testnet.fantom.network/
+ChainID:            0xfa2
+Symbol:             FTM
+```
+
+<br />
+<!-- LICENSE -->
+
+## License
+
+Distributed under the MIT License. See `LICENSE` for more information.
+
+<br />
+
+<!-- Acknowledgements -->
+
+## Acknowledgements
+
+-   [Img Shields](https://shields.io)
+-   [Choose an Open Source License](https://choosealicense.com)
+
+<br />
+
+<!-- Contributions -->
+
+## Contributions
+
+-   eabz
+-   helloitsm3
```

Comment:
Probably will remove also contributors. Git is enough to know who is contributing.